scalyr_agent/util.py

```diff
@@ -2357,3 +2360,19 @@ def summarize(self):
             self.max(),
             self.estimate_median(),
         )
+
+
+def max_ignore_none(*args, **kwargs):
+    """max function which ignores None values"""
```

Comment:
Add documentation for the arguments, especially what the first one (key) is for.